### PR TITLE
getKeys: Fix detection of whether user is logged in to Azure CLI

### PR DESCRIPTION
**Pull request by curtisman**

### CODE REVIEW

**Review by RishhiB — APPROVED** [review]
LGTM!

File: tools/getkeys/index.js [hunk 1]
```diff
@@ -117,7 +117,7 @@ async function execAsync(command, options) {
 class AzCliKeyVaultClient {
     static async get() {
 
-        await execAsync("az account set --subscription Fluid");
+        await execAsync("az ad signed-in-user show");
```
Comment (alexvy86) [inline tools/getkeys/index.js:120]:
```suggestion
        // The 'az keyvault' commands below work regardless of which subscription is currently "in context",
        // as long as the user is listed in the vault's access policy, so we don't need to do 'az account set'.
        await execAsync("az ad signed-in-user show");
```

**Review by alexvy86 — APPROVED** [review]
Just an extra comment for future reference

File: tools/getkeys/index.js (hunk 1 above)
Comment (alexvy86) [reply tools/getkeys/index.js:120]: Ah, too late 😜

**Pull request merged**